### PR TITLE
updated readme for secrets

### PR DESCRIPTION
**Pull request by milanhajek**

### CODE REVIEW

File: src/tools/extended-test-framework/deploy/test_director/README.md [hunk 1]
```diff
@@ -5,12 +5,28 @@ For security reasons we are using sealed secrets approach from [bitnami](https:/
...
+   name: test-director-secret
+   namespace: mayastor-e2e
+   type: Opaque
+   data:
+     xray_client_id: RVENDYxzOURFMkVY=
+     xray_client_secret: MGT3YmIyNmY2OWU4NQ==
+     slack_web_hook: aHRy9o29rcy5zbGckpy=
+     jira_api_token: WlRjBiM0pBYldGNVlXUmhkR0VTFNGRjRTMW80Yl
+     #key: base64-value
+   ```
```
Comment (rohan2794) [inline src/tools/extended-test-framework/deploy/test_director/README.md:28]: can you please add a note on how to get  id, secret, hook and token

**Review by rohan2794 — APPROVED** [review]
One minor commment

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Comment by milanhajek** [comment]
bors merge

**Pull request merged**